invoice/src/main/java/org/killbill/billing/invoice/template/formatters/DefaultInvoiceFormatter.java

```diff
@@ -71,6 +72,18 @@
     private final InternalTenantContext context;
     private final ResourceBundleFactory bundleFactory;
 
+    public static Map<java.util.Currency, Locale> currencyLocaleMap;
```

Comment:
No problem, I'll move it. Not sure what do you mean with add it to the constructor parameters because this method takes possible values from Locale.getAvailableLocales().